labs/shipped-bootstrap-spark/1.md

```diff
@@ -1,43 +1,33 @@
-# Shipped 5 - Bootstrap a Project Using Spark
+# Shipped 5 - Bootstrap Spark Project
 
-You can bootstrap a Shipped project using Spark.
+## Introduction 
+Cisco Spark helps modern teams work together. Sign in to create rooms with group messaging, content sharing, video calling, and desktop sharing.
 
+Cisco Spark provides  us developer platform using rest api with which we have created a sample  application. To know more click on - <a href="https://github.com/CiscoCloud/shipped-buildpack-template-spark" target="_blank">Cisco Spark Buildpack Template</a>.
 
 ## Objectives
-Completion time: 5 minutes
-
 You will learn how to:
-
-- Bootstrap a project using Spark
-
-
-
+- Bootstrap Spark Sample Application using Shipped
+- Completion time : 10 min approx
 
 ## Prerequisites
 
-- *Managing Services* Learning Lab
-
-
-
+- Github Login Account
+- Spark API token to login to our sample Spark app, after bootstrap.
 
 ## References
-You can open these references in a separate tab and refer to them during this Lab.
-
-
-- <a href="https://developer.cisco.com/site/shipped/" target="_blank">Shipped Getting Started Guide</a>  
-- <a href="https://cisco.jiveon.com/docs/DOC-811787" target="_blank">High-level view of Shipped</a> 
-- <a href="https://developer.cisco.com/site/shipped/" target="_blank">Shipped User Guide</a>  
-- <a href="https://ciscoshipped.io/shipped/api-docs/build/index.html" target="_blank">Shipped API Reference</a>  
-- <a href="https://developer.cisco.com/site/shipped/" target="_blank">Shipped CLI Reference</a>  
-
+- Cisco Spark: Shipped Buildpack demo video (10 min), please follow <a href="https://cisco.webex.com/cisco/ldr.php?RCID=98837d3fbf139ce537e8f11a660ccff0" target="_blank" >video link here</a>.
+- Walkthrough Cisco Shipped to create, build and deploy project,<a href="http://shipped-cisco.com/shipped/api-docs/build/index.html#walkthroughs" target="_blank">click here</a>. 
+- <a href="https://github.com/CiscoCloud/Shipped-Learning-Labs/blob/master/labs/shipped-quick-start/1.md" target="_blank">Quick Start with Cisco Shipped</a> reference guide. 
 
 ## To Bootstrap a project with Spark 
 
-1. Create a project.
-2. From the **API** tab, choose the Spark buildpack.
+1. We need to create project in Shipped using Spark buildpack, refer to Getting Started section of Shipped document: <a href="http://shipped-cisco.com/shipped/api-docs/build/index.html#walkthroughs" target="_blank">Click here</a>
 
-	![](posts/files/shipped-bootstrap-spark/assets/add-spark-api.png)
+Note : We need to make a small change at service creation step, just select "Cisco Spark" Buildpack under API tab.
 
+![](assets/add-spark-api.png)
 
-3. Continue building your application as normal.
+Rest project creation and deployment section will remain same, <a href="https://ciscoshipped.io/shipped/api-docs/build/index.html#5.-build-your-project" target="_blank">more detail.</a>
```

Comment:
Is this part of the procedure? 
E.g., Continue to create your project as described in build-your-project.